packages/endpoints-overlay/src/EndpointsOverlay.story.js

```diff
@@ -2,7 +2,8 @@ import BaseMap from "@opentripplanner/base-map";
 import PropTypes from "prop-types";
 import React from "react";
 import { action } from "@storybook/addon-actions";
-import { Cat, Dog } from "styled-icons/fa-solid";
+import { Cat } from "@styled-icons/fa-solid/Cat";
+import { Dog } from "@styled-icons/fa-solid/Dog";
```

Comment:
Hey folks, were you able to confirm that changing to this format for these imports resolves the tree-shaking issues?  I see [here](https://github.com/styled-icons/styled-icons#usage) in their docs they talk about the different import styles, but don't explicitly say that is needed to exclude other icons in a pack from being bundled and elsewhere in the docs it seems to imply that the tree-shaking should work with the previous style you were using

Reply:
Apologies, I wasn't caught up on Slack and now see where this coming from and that you're seeing a reduced bundle after implementing

Reply:
No worries! Yes changing these imports is what resolved the tree-shaking issues.